submissions/chidubemOnuoha/database2/init.sql

```diff
@@ -0,0 +1,44 @@
+--create a database for a transport company
+CREATE DATABASE REFERENCES,
+--run your command to create your database
+
+--we are going to use the databse
+USE transport_company;
+
+CREATE TABLE  passengers (
+    id NOT NULL AUTO_INCREMENT PRIMARY key,
+    full_name VARCHAR NOT NULL,
+    passengers_id INT NOT NULL age VARCHAR,
+    sex VARCHAR NOT NULL, 
+    created_at DATETIME,
+    
+);
+
+--creating a passengers detail table
+ CREATE TABLE passengers_details (
+     id INT NOT NULL AUTO_INCREMENT  FOREIGN KEY (passengers_id),
+     passenger_class VARCHAR NOT NULL,
+     ticket_number VARCHAR NOT NULL,
+     trip_fare VARCHAR NOT NULL,
+     assigned_cabin VARCHAR NOT NULL,
```

Comment:
Recall some of the passengers might not be assigned cabins yet, so this could be null